docs/src/snap/reference/certificates.md

```diff
@@ -0,0 +1,45 @@
+# Cluster Certificates
```

Comment:
```suggestion
# Cluster Certificates and Configuration Directories
```
I wonder if we should split these into two pages but then we'd have to half pages - not ideal. 
Let's rather reflect the second part of this page in the title of the doc and the name of the file.